src/sunstone/etc/sunstone-views/kvm/cloud.yaml

```diff
@@ -3,211 +3,247 @@ link_logo:
-    datastores-tab:
-        table_columns:
-            - 0         # Checkbox
-            - 1         # ID
-            - 2         # Name
-            - 3         # Owner
-            - 4         # Group
-            #- 5        # Capacity
-            #- 6        # Cluster
-            #- 7        # Basepath
-            #- 8        # TM
-            #- 9        # DS
-            #- 10       # Type
-            - 11        # Status
-            #- 12       # Labels
-            #- 13       # Search data
\ No newline at end of file
+      # In the cloud view, delete is the equivalent
+      # of 'onetemplate delete --recursive'
+      Template.delete: true
+      VM.rename: true
+      VM.resume: true
+      VM.reboot: true
+      VM.reboot_hard: true
+      VM.poweroff: true
+      VM.poweroff_hard: true
+      VM.undeploy: false
+      VM.undeploy_hard: false
+      VM.terminate: true
+      VM.terminate_hard: true
+      VM.resize: true
+      VM.disk_resize: true
+      VM.attachdisk: true
+      VM.detachdisk: true
+      VM.disk_saveas: true
+      VM.attachnic: true
+      VM.detachnic: true
+      VM.snapshot_create: true
+      VM.snapshot_revert: true
+      VM.snapshot_delete: true
+      VM.disk_snapshot_create: true
+      VM.disk_snapshot_revert: true
+      VM.disk_snapshot_rename: true
+      VM.disk_snapshot_delete: true
+      VM.save_as_template: true
+      VM.migrate_poff: false
+      VM.migrate_poff_hard: false
+      VM.lockU: true
+      VM.unlock: true
+      VM.startvnc: true
+      VM.startvmrc: true
+      VM.startspice: true
+      VM.vnc: true
+      VM.ssh: true
+      VM.rdp: true
+      VM.save_rdp: true
+      VM.save_virt_viewer: true
+      VM.updateconf: true
+      VM.attachsg: true
+      VM.detachsg: true
+      VM.instantiate_name: true
+      VM.backup_dialog: true
+      VM.backup: true
+    dashboard:
+      # Connected user's quotas
+      quotas: true
+      # Overview of connected user's VMs
+      vms: true
+      # Group's quotas
+      groupquotas: false
+      # Overview of group's VMs
+      groupvms: false
+    create_vm:
+      # True to allow capacity (CPU, MEMORY, VCPU) customization
+      capacity_select: true
+      # True to allow NIC customization
+      network_select: true
+      # True to allow vmgroup customization
+      vmgroup_select: true
+      # True to allow DISK size customization
+      disk_resize: true
+      # True to allow datastore customization
+      datastore_select: true
+  settings-tab:
+    panel_tabs:
+      settings_info_tab: false
+      settings_config_tab: true
+      settings_quotas_tab: true
+      settings_accounting_tab: true
+      settings_showback_tab: true
+    actions:
+      # Buttons for settings_info_tab
+      User.update_password: true
+      User.login_token: true
+      User.two_factor_auth: true
+      # Buttons for settings_config_tab
+      Settings.change_language: true
+      Settings.change_password: true
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded password. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources using this credential and storing future credentials in a management service such as [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).

[Learn more about the use of hardcoded credentials](https://cwe.mitre.org/data/definitions/798.html)